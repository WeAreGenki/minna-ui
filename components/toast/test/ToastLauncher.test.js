'use strict';

const ToastLauncher = require('../src/ToastLauncher.svelte');

describe('ToastLauncher component', () => {
  it('renders correctly with defaults', () => {
    const target = document.createElement('div');
    new ToastLauncher({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
