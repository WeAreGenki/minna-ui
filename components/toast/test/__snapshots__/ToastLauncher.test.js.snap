// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastLauncher component renders correctly with defaults 1`] = `""`;
